ODIN_II/SRC/simulate_blif.cpp

```diff
@@ -3198,7 +3246,8 @@ static nnode_t* print_update_trace(nnode_t* bottom_node, int cycle) {
             for (i = 0; i < node->num_input_pins; i++) {
                 npin_t* pin = node->input_pins[i];
                 nnet_t* net = pin->net;
-                nnode_t* node2 = net->driver_pin->node;
+#warning "Oops this be broke come fix"
```

Comment:
this is failing Travis on compilation warnings, this can be solved with a for loop I believe

Reply:
Sounds good. Will pick that up on Tuesday